Models/TickerModel.cs

```diff
@@ -37,33 +42,10 @@ public TickerModel(string name)
             Random rnd = new Random();
             _price = rnd.Next(0, 1000);
 
-            Task.Run(() => UpdatePrice());
-        }       
+            _priceChanged = Observable.Timer(TimeSpan.FromSeconds(0),
+                TimeSpan.FromMilliseconds(RefreshTime)).Select((x) => rnd.Next(-10, 10));
 
-        /// <summary>
-        /// Loop to update the stock price
-        /// </summary>
-        void UpdatePrice()
-        {
-            Random rnd = new Random();
-            while (!_stop)
-            {
-                int priceChange = rnd.Next(-10, 10);
-                _price += priceChange;
-                OnPriceChanged(priceChange);
-
-                Thread.Sleep(RefreshTime);
-            }
-        }        
-
-        void OnPriceChanged(int priceChange)
-        {
-            PriceChanged?.Invoke(this, priceChange);
-        }
-
-        public void Dispose()
-        {
-            _stop = true;
-        }
+            _priceChanged.Subscribe(x => _price += x);
```

Comment:
This leaks memory. TickerModel will never be collected by GC, because you are not ending your subscriptions (here, and in the VM)